app/grandchallenge/workstations/static/workstations/js/session-region.js

```diff
@@ -3,15 +3,30 @@ function setModalLoadingMessage(msg) {
 }
 
 function ping(url) {
-    let end = null;
+    let errored = false;
+
+    // First try establishes TLS
+    $.ajax({
+        url: url,
+        async: false,
+        cache: false,
+        success: ()=>{},
+        error: function () {
```

Comment:
I'd say either use an arrow function here, or make the success function a normal `function()`